examples/react-native/metro.config.js

```diff
@@ -116,6 +116,12 @@ config.resolver.blockList = [
 
 // point to the example app deps for React and native modules deps
 config.resolver.extraNodeModules = {
+  '@aws-amplify/react-native': path.resolve(
+    __dirname,
+    'node_modules/@aws-amplify/react-native'
+  ),
+  'base-64': path.resolve(__dirname, 'node_modules/base-64'),
```

Comment:
Reason for having to add this: https://github.com/aws-amplify/amplify-js/blob/console-preview/packages/core/src/utils/globalHelpers/index.native.ts#L5